packages/story-editor/src/components/canvas/renderResourcePlaceholder.js

```diff
@@ -26,6 +26,8 @@ const placeholderStyles = css`
   left: 0;
   min-height: 100%;
   min-width: 100%;
+  max-height: 100%;
+  max-width: 100%;
```

Comment:
min and max could be removed if we do:
```jsx
return <BlurhashContainer height="100%" width="100%" hash={blurHash} punch={1} />
```
otherwise, we cannot override the width/height from Blurhash without `!important`.

But because `placeholderStyles` is also used in the `BaseColorContainer` we need to leave something here too (width/height=100%?).

Reply:
It is all the same to me. Changed to use height / width 100%.